plugins/scaffolder-react/src/next/components/Workflow/Workflow.tsx

```diff
@@ -128,8 +128,8 @@ export const Workflow = (workflowProps: WorkflowProps): JSX.Element | null => {
               className={styles.markdown}
               linkTarget="_blank"
               content={
-                description ??
-                sortedManifest.description ??
+                (description && description.trim()) ||
+                sortedManifest.description ||
```

Comment:
The `description` property is not currently being passed, so this doesn't solve it